config.js

```diff
@@ -34,6 +34,12 @@ module.exports = {
   // See `bufferSize`.
   flushDelaySeconds: 30,
 
+  // If true, traces will be published as soon as `bufferSize` traces have
+  // been collected if this occurs before `flushDelaySeconds`.
+  // If false, traces will flush every `flushDelaySeconds` and additional
+  // traces beyond `bufferSize` will be dropped.
+  publishImmediately: true,
+
```

Comment:
I don't like the property name too much as it is not describe what it actually does. `publishOnBufferFull` would be more better, but even that doesn't describe the behaviour that the subsequent traces will be dropped.